system/Cache/Handlers/FileHandler.php

```diff
@@ -94,8 +92,7 @@ public function initialize()
 	 */
 	public function get(string $key)
 	{
-		$key = $this->prefix . $key;
-
+		$key  = static::validateKey($key, $this->prefix);
```

Comment:
If the `$this->prefix` is really not included in the return of `validateKey`, then `$key` should be prepended with `$this->prefix` before it is used.

Applies to other instances where `static::validateKey` is used.

Reply:
You're right! My mistake. If you agree with my other comments on prefixing I will update these.